servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1390,7 +1417,7 @@ private long loadMetadataDirectory(TachyonURI path, boolean recursive)
   }
 
   public boolean mount(TachyonURI tachyonPath, TachyonURI ufsPath)
-      throws FileAlreadyExistsException, InvalidPathException, IOException {
+      throws FileAlreadyExistsException, InvalidPathException, IOException, AccessControlException {
```

Comment:
`mount()` invoke `loadMetadataDirectory()`, and then invoke `mkdir()`, where checking happens.